frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -551,11 +551,17 @@
   "Get Started using applications by choosing a code sample.": "Get Started using applications by choosing a code sample.",
   "Select a Project to view the list of samples<1></1>.": "Select a Project to view the list of samples<1></1>.",
   "Builds are a core concept in OpenShift Container Platform. A build describes a process for transforming source code into a runnable image.": "Builds are a core concept in OpenShift Container Platform. A build describes a process for transforming source code into a runnable image.",
+  "Builds is an extensible build framework based on the Shipwright project, which you can use to build container images on an OpenShift Container Platform cluster.": "Builds is an extensible build framework based on the Shipwright project, which you can use to build container images on an OpenShift Container Platform cluster.",
```

Comment:
subject verb agreement: Builds are*

p.s., I think it would be better to write the full "Builds for OpenShift" or "Builds for Red Hat OpenShift" so that there is slightly less confusion with the other builds option in the dropdown menu

Reply:
Let's leave these decisions to the UX. They are looking into it and will update accordingly :)